<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ES IDE</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-1</y>
      <width>1001</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="editVarTab">
     <attribute name="title">
      <string>Edit Variables</string>
     </attribute>
     <widget class="QPushButton" name="addVarButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Variable</string>
      </property>
     </widget>
     <widget class="QListWidget" name="varList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteVarButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Variable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="varNameEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Existing Variable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Name</string>
      </property>
     </widget>
     <widget class="QListWidget" name="valueList">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>256</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Values</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="varValueEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>240</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteValueButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>170</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Value</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addValueButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>170</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Value</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>577</x>
        <y>290</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Errors</string>
      </property>
     </widget>
     <widget class="QPushButton" name="renameVarButton">
      <property name="geometry">
       <rect>
        <x>557</x>
        <y>100</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Variable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="renameValueButton">
      <property name="geometry">
       <rect>
        <x>823</x>
        <y>240</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Value</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="errorsEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editRulTab">
     <attribute name="title">
      <string>Edit Rules</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionCheck"/>
    <addaction name="actionInterpret"/>
    <addaction name="actionGenerate_Source_Code"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionInterpret">
   <property name="text">
    <string>Interpret</string>
   </property>
  </action>
  <action name="actionGenerate_Source_Code">
   <property name="text">
    <string>Generate Source Code</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
